stl/inc/charconv

```diff
@@ -152,52 +152,55 @@ _NODISCARD to_chars_result _Integer_to_chars(
         return {_Last, errc::value_too_large};
     }
 
-    _CSTD memcpy(_First, _RNext, static_cast<size_t>(_Digits_written));
+    _Copy_n_unchecked4(_RNext, static_cast<size_t>(_Digits_written), _First);
 
     return {_First + _Digits_written, errc{}};
 }
 
-inline to_chars_result to_chars(char* const _First, char* const _Last, const char _Value, const int _Base = 10) noexcept
+_CONSTEXPR23 to_chars_result to_chars(
+    char* const _First, char* const _Last, const char _Value, const int _Base = 10) noexcept
 /* strengthened */ {

```

Comment:
Style: It appears that this is a situation where there would be sufficient room to put the comment on the same line, but clang-format has "sticky" behavior - i.e. it won't do so automatically, but will preserve it after it's done manually. Occurs below.